client.go

```diff
@@ -243,6 +250,11 @@ func (uc Client) IsEnabled(feature string, options ...FeatureOption) (enabled bo
 			// TODO: warnOnce missingStrategy
 			continue
 		}
+
+		staticContext := &context.Context{AppName: uc.options.appName, Environment: "default"}
```

Comment:
We shouldn't need to create this every time because the "static" part of this will never change once the client has been initialized. Maybe a better solution would be that the `Client` contains a `staticContext` field which gets initialized only once?